### ISSUE TITLE
String compression

### ISSUE THREAD

**Issue by norberte** [ubco/week-visualizer#1]
Here is a string compression that I found online : http://stackoverflow.com/questions/3649485/how-to-compress-a-string

Take a look at it, since it is a different way of approaching the compression.

**Comment by norberte** [comment]
Kevin, did you see my Java file commit ? Would that algorithm work if we would write it in Javascript ?
It is the same thing that you are trying to hard-code, but this is done by streams and libraries... All I am saying that it accomplishes the purpose and it is shorter and easier.

Reply to this if you read it.
Thanks, Norbert

**Comment by vankessel** [comment]
Hey Norbert, I saw it, didn't look into the details of how it works but I'm
sure we could implement it in Javascript.
The reason I committed mine is because I was already halfway done and want
to see if it would work. I'm fine if we use the Java one though.

**Comment by vankessel** [comment]
I don't think we can use that string compression. I tried it out and some of the characters created can not be copy-pasted.
